### ISSUE TITLE
Needs editing for Python 3.x

### ISSUE THREAD

**Issue by RobertBarrow** [zopefoundation/ZODB#200]
My version (working in Python 3.6.1) for reference:
https://github.com/RobertBarrow/python3zodb/blob/master/employees.py

**Comment by jamadden** [comment]
Do you mean the code snippets contained in doc/articles/ZODB1.rst?

**Comment by RobertBarrow** [comment]
Yes, sorry.  The code snippets are based on Python 2.  I had to tweak them to work on Python 3.
